lib/updateArtifact.js

```diff
@@ -1,15 +1,14 @@
 var readAllNetworkConfigs = require('./readAllNetworkConfigs')
 var mostRecentEntryByContractName = require('./mostRecentEntryByContractName')
 
-module.exports = async function (artifact, networkConfigs) {
-  if (!networkConfigs) {
-    networkConfigs = await readAllNetworkConfigs()
-  }
-  var { contractName } = artifact
+module.exports = async function (network, networkConfigs) {
```

Comment:
`network` should be named `artifact`, because it's updating the artifact by merging in the new network config.